### ISSUE TITLE
Select Icons Automatically 

### ISSUE THREAD

**Issue by Gammachange** [beemdevelopment/Aegis#1534]
See here:
https://github.com/beemdevelopment/Aegis/issues/509#issue-651101670

**Comment by Gammachange** [comment]
You're right, sorry, I didn't quite understand the message here. https://github.com/beemdevelopment/Aegis/issues/509#issue-651101670
I'm sorry, that's my fault.

**Comment by michaelschattgen** [comment]
No worries! I was just wondering. I actually already started working on assigning/suggesting an icon from your installed icon pack when adding a new entry. I assume that's what you want, right? Have you already tried one of our icon packs?

**Comment by Gammachange** [comment]
That's good to hear (yes I've tried several of them). Thank you very much, I am happy to be part of your wonderful community!

**Comment by afterdark1880** [comment]
@michaelschattgen Your Android app is wonderful and it speaks to those who are concerned with privacy and security. 
For now, I've switched over to 2FAS 'cause it automatically assigns icons, even though the overall user experience is far simpler and less customizable, settings-wise.
Not having icons ready at hand is a major drawback, as they're very useful when visually searching through dozens of services.
I'll be happy to go back to your app when you make it automatic.
Have a nice day 😉
